test/Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests/KafkaE2EFixture.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.apps.brokers;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.apps.languages;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.apps.type;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.cleanup;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.command;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.command.queue;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.initializer;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.process;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.queue;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.queue.eventhub;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.queue.operation;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.queue.storageQueue;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.Util;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests
+{
+    public class KafkaE2EFixture : IAsyncLifetime
+    {
+        private Language language;
+        private AppType appType;
+        private BrokerType brokerType;
+        protected bool isInitialized = false;
+
+        public KafkaE2EFixture()
+        {
+            Console.WriteLine("Kafkae2efixture");
+        }
+
+        public void OrchestrateInitialization()
+        {
+            if(!isInitialized)
```

Comment:
nit comment : if we change this to :-
if(isInitialized) 
  return;
and remove the if condition and keep initialization as a free code

Reply:
Done.